operators/src/source/gdal_source.rs

```diff
@@ -254,6 +254,11 @@ where
         let rasterband: GdalRasterBand =
             dataset.rasterband(data_set_params.rasterband_channel as isize)?;
 
+        // get the no_data_value. TODO: move to dataset parameters? Keep in mind, that this can change for each time step.
+        // We also need to ge metadata from the dataset (for each tile) e.g. scale + offset.
```

Comment:
what do you mean by that? To turn raw data into a measurement? Why don't we use the expression operator?

Also type: "ge" -> "get"

Reply:
yes i don't want the source to do the scaling but e.g. for meteosat there are offset and scale values which are different for each timestep. will we store them in the database too?

Reply:
that's an interesting case I wasn't aware of. First of all I would transform the time series once into the measurement instead of doing it every time. But of course we want to be able to do that in geo engine. We either need to enhance the raster tile meta data to carry this meta data and use it in the expression or we need some raster - datacollections join where we get the scaling info from somewhere and use it in the operator.

Reply:
i think that is not a good idea. First, this will blow up the required storage space. We will never store all the transformed MSG data. The wavelet encoded original data is already ~100TB large. And second: sometimes i want the real raw data and somteimes i want to transform it.

The RasterTile needs a map for metadata that is correct. Do you want to store the metadata in the database? Where will it come from? Also if you want to transform the data once, we need to read and transform it... So we have to read the metadata in any case and we need to handle this somewhere.

Reply:
I would store them in the database if they are unique for each tiff file and otherwise compute them in the `loading_info` method based on some meta data in the database.

Reply:
but who inserts them into the database?